### PR TITLE
BUGFIX: BB-1481 Pivot grouping turned off by default

### PR DESCRIPTION
**Pull request by petrjanu**

### CODE REVIEW

File: docs/pivot_table_component.md [hunk 1]
```diff
@@ -238,7 +238,7 @@ const config = {
 | filters | false | [Filter[]](filter_visual_components.md) | An array of filter definitions |
 | config | false | [ConfigObject](#configuration-menu) | The configuration object |
 | sortBy | false | [SortItem[]](result_specification.md#sorting) | An array of sort definitions |
-| groupRows | false | boolean | Specifies whether [grouping of the same values in attribute columns](#grouping) is enabled (`true`; default) or disabled (`false`). | 
+| groupRows | false | boolean | Specifies whether [grouping of the same values in attribute columns](#grouping) is enabled (`true`) or disabled (`false`; default). |
```
Comment (evgeniabogranskaya) [inline docs/pivot_table_component.md]: Just a small suggestion:

Specifies whether [grouping of the same values in attribute columns](#grouping) is disabled (`false`; default) or enabled (`true`).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/44/ : SUCCESS in 43s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/58/ : SUCCESS in 1m 31s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/46/ : SUCCESS in 37s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/60/ : SUCCESS in 1m 15s

File: docs/pivot_table_component.md [hunk 2]
```diff
@@ -171,16 +171,15 @@ const sortBy = [
 
 ## Grouping
 
-By default, the same values in column attributes are grouped across the rows, which helps improve data readability in pivot tables. The header of the grouped columns is displayed at the top of the table even when you scroll through a long group.
+The same values in column attributes are grouped across the rows, which helps improve data readability in pivot tables. The header of the grouped columns is displayed at the top of the table even when you scroll through a long group.
 
-The grouping is applied when the following conditions are both met:
+The grouping is applied when the following conditions are all met:
+* The property `groupRows` is set to `true` (see [Properties](#properties)).
```
Comment (evgeniabogranskaya) [inline docs/pivot_table_component.md]: The `groupRows` property is set to `true` (see [Properties](#properties)).

File: docs/pivot_table_component.md [hunk 3]
```diff
@@ -171,16 +171,15 @@ const sortBy = [
 
 ## Grouping
 
-By default, the same values in column attributes are grouped across the rows, which helps improve data readability in pivot tables. The header of the grouped columns is displayed at the top of the table even when you scroll through a long group.
+The same values in column attributes are grouped across the rows, which helps improve data readability in pivot tables. The header of the grouped columns is displayed at the top of the table even when you scroll through a long group.
 
-The grouping is applied when the following conditions are both met:
+The grouping is applied when the following conditions are all met:
```
Comment (evgeniabogranskaya) [inline docs/pivot_table_component.md]: The grouping is applied when the following conditions are **all** met:

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/47/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/61/ : SUCCESS in 51s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/18/ : SUCCESS in 32s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/63/ : SUCCESS in 1m 17s

**Pull request merged**